core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -51,13 +55,38 @@ angular.module('oppia').controller(
         return `${contentType} section of "${stateName}" card`;
       };
 
+      var _getThreadHandlerUrl = function(suggestionId) {
+        return UrlInterpolationService.interpolateUrl(
+          '/threadhandler/<suggestionId>', { suggestionId });
+      };
+
+      var _getThreadMessagesAsync = function(threadId) {
+        return $http.get(_getThreadHandlerUrl(threadId)).then((response) => {
+          let threadMessageBackendDicts = response.data.messages;
```

Comment:
Prefer `const`? The contents of the map changes, but the name is not re-assigned.

Reply:
Yeah we can do that, though it's not mandatory in the codebase, I'll fix it in a separate PR, will that be fine?